coreldraw_cdr.ksy

```diff
@@ -2655,3 +3120,70 @@ types:
             - cdrSVGPalette
             - SVGColor # file name (SVGColor.xml)
           doc: SVG Colors
+  text_style_flags:
+    seq:
+      - id: emphasis_raw
+        type: u2
+      - id: unknown
+        type: b2
+      - id: underline
+        type: b3
+        enum: decoration
+      - id: overline
+        type: b3
+        enum: decoration
+      - id: strike_through_line
+        type: b3
+        enum: decoration
+      - id: script
+        type: b2
+        enum: script_type
+      - type: b3
+    instances:
+      italic:
+        value: emphasis_raw == 0x0080 or emphasis_raw == 0x2000
+      bold:
+        value: emphasis_raw == 0x1000 or emphasis_raw == 0x2000
+    enums:
+      decoration:
+        0: none
+        1: single_thin
+        2: single_thin_word
+        3: single_thick
+        4: single_thick_word
+        5: double_thin
+        6: double_thin_word
+      script_type:
+        0: none
+        1: superscript
+        2: subscript
```

Comment:
https://community.coreldraw.com/sdk/api/draw/17/e/cdrfontposition:

> **Members:**
> 
> Name | Value | Description
> -- | -- | --
> cdrNormalFontPosition | 0 | Specifies normal font position
> cdrSubscriptFontPosition | 1 | Specifies subscript font position
> cdrSuperscriptFontPosition | 2 | Specifies superscipt font position
> cdrMixedFontPosition | 3 | Specifies mixed font position

Hmm, it looks like subscript and superscript have swapped values in the API, but I saw that your enum is correct because it matches sample files.

```suggestion
      # NOTE: https://community.coreldraw.com/sdk/api/draw/17/e/cdrfontposition has subscript at 1
      # and superscript at 2, but that doesn't agree with sample files - the interpretation below
      # does.
      script_type:
        0: none
        1: superscript
        2: subscript
```